v3/src/components/graph/utilities/graph-utils.ts

```diff
@@ -187,12 +193,13 @@ export function percentString(value: number) {
 
 export const lsrlEquationString = (
   slope: number, intercept: number, attrNames: {x: string, y: string}, caseValues: Point[],
-  rSquared?: number, confidenceBandsEnabled?: boolean, color?: string, interceptLocked=false
+  confidenceBandsEnabled?: boolean, rSquared?: number, color?: string, interceptLocked=false, sumOfSquares?: number
 ) => {
   const float = format(".3~r")
   const floatIntercept = format(".1~f")
   const floatSeSlope = format(".3~f")
   const linearRegression = leastSquaresLinearRegression(caseValues, interceptLocked)
+  const floatSumOfSquares = format(",.0f")
```

Comment:
Use named constants for these format specifiers so if we need to change them they can be changed in one place. I'm concerned that these fixed-decimal place formats won't work well for some data sets. How does v2 handle formatting these values?